tests/phpt/dl/1036_to_array_debug_shape_nullable.php

```diff
@@ -0,0 +1,16 @@
+@ok
+<?php
+require_once 'kphp_tester_include.php';
+
+function to_array_debug_shape_nullable() {
+  /** @var shape(foo:string, bar?:int) */
+  $shape = shape(["foo" => "baz"]);
+  $dump = to_array_debug($shape);
+
+  #ifndef KPHP
+  $dump = ['foo' => 'baz', 'bar' => null];
+  #endif
+  var_dump($dump);
+}
```

Comment:
Please, add more tests:
1) an instance having two shape fields 
2) an instance as a value of one of the keys
3) a shape within a shape

Reply:
We discussed that such cases more or less covered in other tests